src/app/zap-templates/zcl/data-model/chip/thermostat-cluster.xml

```diff
@@ -106,15 +110,96 @@ limitations under the License.
     <item fieldId="2" name="CoolSetpoint" type="int16s" isNullable="true"/>
   </struct>
 
+  <bitmap name="PresetTypeFeaturesBitmap" type="bitmap8">
+    <cluster code="0x0201"/>
+    <field name="Automatic" mask="0x01"/>
+    <field name="SupportsNames" mask="0x02"/>
+  </bitmap>
+
+  <enum name="PresetScenarioEnum" type="enum8">
```

Comment:
Spec does not say what this is derived from.  Please make sure a spec issue is filed and referenced here.